boot/setup.sh

```diff
@@ -162,6 +169,7 @@ manta_common_presetup
 echo "Adding local manifest directories"
 manta_add_manifest_dir "/opt/smartdc/mako"
 manta_add_manifest_dir "/opt/smartdc/minnow"
+manta_add_manifest_dir "/opt/smartdc/rebalancer-agent"
```

Comment:
I think we should drop this, but it is fine either way. It isn't needed because rebalancer-agent doesn't have a sapi_manifests dir.  Having it listed will result in config-agent logging a warning or error that it cannot find "/opt/ssmartdc/rebalancer-agent/sapi_manifests", but otherwise config-agent will be just fine.

Reply:
Yep, fair enough, I'll drop that.